crates/runtime/src/table.rs

```diff
@@ -170,9 +167,9 @@ impl Table {
         Ok(Table::Static { data, size, ty })
     }
 
-    fn limit_new(plan: &TablePlan, limiter: Option<&mut dyn ResourceLimiter>) -> Result<()> {
-        if let Some(limiter) = limiter {
-            if !limiter.table_growing(0, plan.table.minimum, plan.table.maximum) {
+    fn limit_new(plan: &TablePlan, store: &mut Option<&mut dyn Store>) -> Result<()> {
```

Comment:
Similar to `Memory` I think we'll want this to be `&mut dyn Store` instead of an `Option`